### ISSUE TITLE
Cannot follow symlinks on vagrant encapsulated flynt dev environment

### ISSUE THREAD

**Issue by Livepixelz** [flyntwp/flynt-starter-theme#266]
Hello team !

First of all thanks for all the work you've done to make wordpress development easier. 👍 🥇  💯

I have an issue while i'm trying to setup my flynt dev environment using Trellis as you advise it on the "Getting started" page.
The problem is that after i installed it all and launched my dev site, here is the issue that i have :
```
Fatal error: Uncaught Twig_Error_Loader: Unable to find template "/srv/www/{mysitedomainname}/current/web/app/themes/livepixelz/dist/Components/NavigationMain/index.twig" 
```
I double checked the path, which is correct... My guess is that maybe Vagrant doesn't support symbolic links, as there are some in that very file path.

Have you folks encountered the same issue in your development setup journey ?

Thanks for your answers.
🙏

**Comment by Qakulukiam** [comment]
Hey there,

thank you for your kind words! I will take a look at this issue and see if I can reproduce it. It might be related to #235 which has a PR (#265) currently being evaluated.

**Comment by Qakulukiam** [comment]
After some tests with a quick Trellis setup, I have been able to reproduce a similar problem:

Although I don't get the error message, none of the components are being rendered when using Flynt with Trellis. After applying the fix in #265 however, it works fine for me.

Let me know, if that works for you as well.

**Comment by Qakulukiam** [comment]
Merged into master with v0.4.0
